docs/en/sql-reference/functions/hash-functions.md

```diff
@@ -1777,34 +1777,66 @@ Result:
 └────────────────────────────────────────────────────────────────────────┘
 ```
 
-## sqid
+## encodeSqid
 
-Transforms numbers into a [Sqid](https://sqids.org/) which is a YouTube-like ID string.
+Encodes numbers as a [Sqid](https://sqids.org/) which is a YouTube-like ID string.
 The output alphabet is `abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789`.
 Do not use this function for hashing - the generated IDs can be decoded back into numbers.
 
 **Syntax**
 
 ```sql
-sqid(number1, ...)
+encodeSqid(number1, ...)
 ```
 
+Alias: `sqid`
+
 **Arguments**
 
 - A variable number of UInt8, UInt16, UInt32 or UInt64 numbers.
 
 **Returned Value**
 
-A hash id [String](/docs/en/sql-reference/data-types/string.md).
+A sqid [String](/docs/en/sql-reference/data-types/string.md).
+
+**Example**
+
+```sql
+SELECT encodeSqid(1, 2, 3, 4, 5);
+```
+
+```response
+┌─encodeSqid(1, 2, 3, 4, 5)─┐
+│ gXHfJ1C6dN                │
+└───────────────────────────┘
+```
+
+## decodeSqid
+
+Decodes a [Sqid](https://sqids.org/) into numbers.
+
+**Syntax**
+
+```sql
+decodeSqid(number1, ...)
```

Comment:
It decodes from `number` to `string` :nerd_face:

Reply:
Ah, of course ... fixed, thanks.